apps/constellation/settings.cpp

```diff
@@ -89,7 +87,7 @@ Settings::Settings()
 bool Settings::Update(int argc, char **argv)
 {
   UpdateFromEnv("CONSTELLATION_");
-  UpdateFromArgs(argc, argv);
+  UpdateFromArgs(argc, argv, []() { std::exit(1); });
```

Comment:
My preference would be to get update from args to return a `bool` and handle this exit logic from here